components/org.wso2.carbon.identity.oauth2.grant.rest.core/src/main/java/org/wso2/carbon/identity/oauth2/grant/rest/core/RestAuthenticationServiceImpl.java

```diff
@@ -1073,4 +1080,34 @@ private String getFederatedIdpChannelName(String authenticatorName, RestAuthenti
         return null;
     }
 
+    /**
+     * This method is used to get user's channel identifier attribute.
+     *
+     * @param userId                          User's user id.
+     * @param userTenantId                    User's tenant id.
+     * @param requestedClaimURI               Required channel Identifier's claim URI.
+     * @throws AuthenticationServerException  Returns an AuthenticationException.
+     * @return String                         User's chanel Identifier's value.
+     */
+    private String getUserVerificationChannel(String userId,int userTenantId, String requestedClaimURI)
```

Comment:
```suggestion
    private String getUserVerificationChannel(String userId, int userTenantId, String requestedClaimURI)
```

Reply:
fixed in [f55a7dd](https://github.com/wso2-extensions/identity-oauth2-grant-rest/commit/f55a7dd7574cc345a359829323b652ce3c8b9202)